packages/backend/src/openapi/webhooks.yaml

```diff
@@ -0,0 +1,275 @@
+openapi: 3.1.0
+info:
+  title: Rafiki Webhooks
+  version: 1.0.0
+webhooks:
+  incomingPaymentCompleted:
+    post:
+      requestBody:
+        description: Notify account servicing entity that an incoming payment was completed and funds should now be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/incomingPayment'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+  incomingPaymentExpired:
+    post:
+      requestBody:
+        description: Notify account servicing entity that an incoming payment has expired and funds should now be withdrawn.
```

Comment:
For an expired incoming payment -> my initial thought was that nothing should happen: since no money was moved?

Reply:
It could have received partial funds. Or it could have been an incoming payment without incoming amount that was still "waiting" for some payments. The WM via OP use case comes to mind.